amx-deps/src/Makefile

```diff
@@ -3,16 +3,16 @@
 ASM = nasm
 CC = gcc
 LD = gcc
-PROG = king.so

```

Comment:
Is there a reason to rename this from king to pawn?

Reply:
**king** seems to be meaningless or unsuitable as a '**AMX Compatibility Layer**' module name... (like who is that king ??)